SLHCUpgradeSimulations/Geometry/test/README.Phase2PixelNtuple.md

```diff
@@ -0,0 +1,66 @@
+This is a short description of the content of the ntuple produced by Phase2PixelNtuple.cc
```

Comment:
please actually use markdown in this file so it is readable

Reply:
hi @emiglior , I guess this comment should be adressed , thanks.

Reply:
@kpedro88 sorry for the slow reaction... any suggestion, reference, cmssw example for md style is welcome